tests/Fixtures/Integration/misc/PHP7_0.test

```diff
@@ -75,14 +75,10 @@ class Foo implements FooInterface
     }
 
     // scalar typehinting
-    public function bar1(int $a): string
-    {
-    }
+    public function bar1(int $a): string {}
```

Comment:
Why do integration tests need to be updated? Doesn't the new option ignore multi line empty bodies?

Reply:
The new option indeed does ignore it. 

But those tests in fact have inlined `foo() {}` in the `INPUT` section, so I adjusted the `EXPECT` section accordingly.